test_cases/Menu.js

```diff
@@ -0,0 +1,116 @@
+export default (folderName, Common) => {
+    describe("Menu", () => {
+      it("Exists", () => {
+        cy.get(
+          '[data-cy="tab-Menu-773dca98-87e0-4dc2-b528-89ec7c98c448-b52e6e96-5033-4c7f-a104-29bd5ddcac4a"]'
+        )
+          .click();
+
+        cy.get(
+          '[data-cy="menu-item KitchenHome cb77ced0-a803-46b7-8a79-f9084d75d51c eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+        )
+          .should("include.text", "Home");
+
+        cy.get(
+          '[data-cy="menu-item 1 93f39714-7d1b-4375-b5b2-89c94a3e96bd eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+        )
+          .should("include.text", "1");
+
+        cy.get(
+          '[data-cy="menu-item Carousel e56d4ad0-d879-43c0-934a-a4004fcc7579 eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+        )
+          .should("include.text", "Carousel");
+
+        cy.get(
+            '[data-cy="menu-item Chart 41d39ac7-e7d9-405d-a570-b79686ccdd5a eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+          )
+            .should("include.text", "Chart");
+
+        cy.get(
+            'div[view_id^="ABViewContainer_"]'
+          )
+            .should("include.text", "Text2")
+            .should("include.text", "Text3");
+      });
+
+      it("Menu Carousel", () => {
+        cy.get(
+          '[data-cy="tab-Menu-773dca98-87e0-4dc2-b528-89ec7c98c448-b52e6e96-5033-4c7f-a104-29bd5ddcac4a"]'
+        )
+          .click();
+
+        cy.get(
+          '[data-cy="menu-item Carousel e56d4ad0-d879-43c0-934a-a4004fcc7579 eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+        )
+          .click();
+        
+        cy.get('[data-cy="carousel 54827db6-497b-43ae-96f9-153b63a9c977"]')
+          .should("include.text", "Default image");
+      });
+
+      it("Menu Chart", () => {
+        Common.RunSQL(cy, folderName, ["add_testkcs2-Menu.sql"]);
+
+        cy.get(
+          '[data-cy="tab-Menu-773dca98-87e0-4dc2-b528-89ec7c98c448-b52e6e96-5033-4c7f-a104-29bd5ddcac4a"]'
+        )
+          .click();
+
+        cy.get(
+          '[data-cy="menu-item Chart 41d39ac7-e7d9-405d-a570-b79686ccdd5a eac2cff7-5069-4f74-9bd9-523a3b633346"]'
+        )
+          .click();
+
+        cy.get('div[view_id^="ABViewChartPie_"]')
+          .should("include.text", "7")
```

Comment:
I think you can simplify this to `.should('exist')`. We don't want to test the Chart Widget here.